api/v1beta1/glanceapi_webhook.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+// GlanceAPIDefaults -
+type GlanceAPIDefaults struct {
+	ContainerImageURL string
+}
+
+var glanceAPIDefaults GlanceAPIDefaults
```

Comment:
Right, in the `Default()` function I should have:
```golang
if spec.GlanceAPITemplate.ContainerImage == "" {
    spec.GlanceAPITemplate.ContainerImage = glanceAPIDefaults.ContainerImageURL
}
```

instead of `glanceDefaults.ContainerImageURL`

Reply:
Done